resources/istio/files/istio-operator-minikube-evaluation.yaml

```diff
@@ -312,6 +308,7 @@ spec:
       replicaCount: 1
       traceSampling: 1
     sidecarInjectorWebhook:
+      useLegacySelectors: true
```

Comment:
shouldn't we use https://istio.io/latest/blog/2021/discovery-selectors/#discovery-selectors-in-action instead?

Reply:
during daily we have agreed to leave it for now and try to remove it in followup task.